db/migrate/20151109120230_drop_old_public_website_tables.rb

```diff
@@ -0,0 +1,37 @@
+class DropOldPublicWebsiteTables < ActiveRecord::Migration
+  def up
+    old_tables = %w( action_items
+        divorce_and_separation_tool_states
+        external_actions
+        isa_alerts
+        page_views
+        partner_polling_partners
+        partner_polling_audit_records
+        partner_polling_poll_users
+        partner_polling_polls
+        partner_polling_redirections
+        partner_polling_votes
+        partner_polling_widget_hosts
+        partner_polling_widgets
+        partners_agreements
```

Comment:
this can be deletec, but this is curious because there is an update table with the same structure of the partner_agreements (it is the agreements_agreements). Since the agreements_agreements table use Active Record STI, you can remove the partner_agreements without problem.